apps/src/applab/ImportScreensDialog.jsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable react/no-danger */
+/* eslint no-unused-vars: "error" */
```

Comment:
:+1: How far are we from enabling this globally?  I'd almost rather to that (in another PR of course) and disable it as an exception, rather than the other way around.